core/src/main/kotlin/fr/sncf/osrd/standalone_sim/StandaloneSimulation.kt

```diff
@@ -198,7 +198,7 @@ fun buildFinalEnvelope(
     scheduledPoints: List<SimulationScheduleItem>
 ): Envelope {
     fun getEnvelopeTimeAt(offset: Offset<TravelledPath>): Double {
-        return provisionalEnvelope.interpolateTotalTimeClamp(offset.distance.meters)
+        return provisionalEnvelope.interpolateDepartureFromClamp(offset.distance.meters)
```

Comment:
The whole function should probably be rewritten with better stop semantics. I *suspect* there's a bug in there but I can't quite put my finger on it. I could do that in a different PR

Reply:
Issue opened https://github.com/OpenRailAssociation/osrd/issues/7749